src/GitHub.Api/Installer/GitInstaller.cs

```diff
@@ -25,10 +25,13 @@ class GitInstallDetails
 
         private readonly bool onWindows;
 
-        public GitInstallDetails(NPath targetInstallPath, bool onWindows)
+        public GitInstallDetails(NPath applicationDataPath, bool onWindows)
         {
             this.onWindows = onWindows;
-            var gitInstallPath = targetInstallPath.Combine(ApplicationInfo.ApplicationName, PackageNameWithVersion);
+
+            PluginDataPath = applicationDataPath.Combine(ApplicationInfo.ApplicationName);
```

Comment:
I think we already have this in the environment, UserCachePath?

Reply:
👍